pkg/kademlia/routing.go

```diff
@@ -91,36 +91,19 @@ func (rt *RoutingTable) CacheSize() int {
 
 // GetBucket retrieves the corresponding kbucket from node id
 // Note: id doesn't need to be stored at time of search
-func (rt *RoutingTable) GetBucket(id storj.NodeID) (bucket dht.Bucket, ok bool) {
+func (rt *RoutingTable) GetBucket(id storj.NodeID) ([]*pb.Node, bool) {
```

Comment:
Why change this to return a `[]*pb.Node` instead of a Bucket type?

Reply:
the ticket is to clean up the bucket interface as it is not used

Reply:
No, the bucket interface is still around and being used. The implementation at pkg/kademlia/bucket.go was not being used

Reply:
https://github.com/storj/storj/pull/898/files#diff-10d55b8cd7e4b1ce8e2c34ca4f5d2ea9L49

Reply:
I'm ok with getting rid of the interface but it's still in the code and `GetBucket` seems like it should return a bucket type. Maybe change the name to be `GetNodes` ?